vendor/assets/stylesheets/schedule/schedule.css

```diff
@@ -32,12 +32,23 @@ grid-day {
   float: left;
 }
 
+day-label {
+  color: #777777;
+  display:block;
+  margin: 0 auto;
+  text-align: center;
+  /* should be the same as grid-hour */
+  border-bottom: 1px solid #e7e7e7;
+  height:16px;
```

Comment:
Didn't realize this is what you were talking about. Can these not be in terms of ```em```?